view/templates/manifest.tpl

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "{{$title}}",
+  "start_url": "{{$baseurl}}",
+  "display": "standalone",
+  "description": "A Decentralized Social Network",
+  "icons": [{
+    "src": "{{$baseurl}}/{{$touch_icon}}",
+    "type": "image/png"
```

Comment:
Does it have to be PNG? What if the config is set to a JPEG/GIF/SVG/BMP icon? Can you please check the MIME type of the provided image?